plugins/org.jkiss.dbeaver.ext.greenplum/src/org/jkiss/dbeaver/ext/greenplum/model/GreenplumDataSource.java

```diff
@@ -61,6 +61,10 @@ public void initialize(@NotNull DBRProgressMonitor monitor) throws DBException {
                 if (matcher.find()) {
                     gpVersion = new Version(matcher.group(1));
                 }
+                matcher = Pattern.compile("Cloudberry Database").matcher(versionStr);
+                if (matcher.find()) {
+                    gpVersion = new Version(7, 0, 0);
```

Comment:
Is there really no way of obtaining the "real" server version?

Reply:
Well, it clearly has the version: `Cloudberry Database 1.0.0 build dev`. Although `7.0.0` is the version of the database it's based on, it's not the version of the Cloudberry database itself.

Reply:
+1. We need to return Cloudberry version (which is 1.0)